tests/integration/dropdown/test_visibility.py

```diff
@@ -0,0 +1,46 @@
+from dash import Dash
+from dash_core_components import Dropdown
+from dash_html_components import Div
+from dash_table import DataTable
+
+import pytest
+
+
+@pytest.mark.DCC788
+def test_visi001_fixed_table(dash_duo):
```

Comment:
```suggestion
def test_ddvi001_fixed_table(dash_duo):
```
so this test is still scoped under dropdowns.